### PR TITLE
changed ant build to compile always with debug=true

### PR DESCRIPTION
**Pull request by denniskniep**

### CODE REVIEW

File: build/build.xml [hunk 1]
```diff
@@ -69,11 +69,8 @@
 	<target name="compile" depends="init" description="compile the source">
 		<echo message="Compiling the source..." />
 		
-		<!-- Compile with debug information if the property "javac.debug" is set to true -->
 		<local name="debug" />
-		<condition property="debug" value="true" else="false">
-			<istrue value="${javac.debug}" />
```
Comment (thc202) [inline build/build.xml]: The property `javac.debug` should be removed from other targets.

File: build/build.xml [hunk 2]
```diff
@@ -69,11 +69,8 @@
...
-		</condition>
+		<property name="debug" value="true" />
```
Comment (thc202) [inline build/build.xml]: The property could be dropped (and set the value directly in the task attribute)?

**Comment by thc202** [comment]
Thank you!

**Comment by kingthorin** [comment]
Thanks

**Pull request merged**